src/main/java/com/taosdata/jdbc/rs/RestfulConnection.java

```diff
@@ -36,6 +38,42 @@ public RestfulConnection(String host, String port, Properties props, String data
         this.metadata = new RestfulDatabaseMetaData(url, props.getProperty(TSDBDriver.PROPERTY_KEY_USER), this);
     }
 
+    /**
+     * A convenient constructor for cloud user
+     * @param url TDengine Cloud URL
+     * @param token TDengine Cloud Token
+     * @throws Exception
+     */
+    public RestfulConnection(String url, String token) throws Exception {
+        super(new Properties());
+        clientInfoProps.setProperty(TSDBDriver.PROPERTY_KEY_TIMESTAMP_FORMAT, String.valueOf(TimestampFormat.TIMESTAMP));
+
+        int poolSize = Integer.parseInt(HttpClientPoolUtil.DEFAULT_MAX_PER_ROUTE);
+        HttpClientPoolUtil.init(poolSize, true);
+        if (url == null || token == null) {
+            throw new Exception("url and token can't be null");
+        }
+        if (!url.startsWith("http://") && !url.startsWith("https://")) {
+            System.out.println("No protocol specified in url, use \"http://\" by default");
```

Comment:
I do not  think System.out is suitable for this connector. you can just set http or add logback or log2j